### PR TITLE
add Sosse software

### PR DESCRIPTION
**Pull request by biolds**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.

### CODE REVIEW

File: software/sosse.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "Sosse"
+website_url: "https://gitlab.com/biolds1/sosse"
```
Comment (KieranRobson) [inline software/sosse.yml:2]:
```suggestion
website_url: "https://sosse.readthedocs.io/en/stable/"
```

File: software/sosse.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+source_code_url: "https://gitlab.com/biolds1/sosse"
+description: "Selenium based open source search engine, offline archiving."
```
Comment (KieranRobson) [inline software/sosse.yml:4]: > https://gitlab.com/biolds1/sosse

```suggestion
description: "Selenium based search engine and crawler with offline archiving."
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Some suggestions below:

**Comment by KieranRobson** [comment]
> https://gitlab.com/biolds1/sosse/-/tags/v1.0.0

I found that 1.0.0 was released 4 months ago, to the day.  If this is to remain open, the suggestions above are still recommended.

**Comment by biolds** [comment]
Hi,

Thanks for your comments! I have updated the MR.

File: software/sosse.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+description: "Selenium based search engine and crawler with offline archiving."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Python
+  - Docker
+tags:
+  - Search Engines
+  - Archiving and Digital Preservation (DP)
+depends_3rdparty: no
```
Comment (Rabenherz112) [inline software/sosse.yml:13]:
```suggestion
```
By default, `depends_3rdparty` is set to "no", so there's no need to set it explicitly to "no" (Can be omitted).

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
Other then this, it looks fine to me.

File: software/sosse.yml (hunk 3 above)
Comment (biolds) [reply software/sosse.yml]: Done, thanks!

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Review by KieranRobson — APPROVED** [review]

**Pull request merged**